applications/rubintv/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rubintv.fullname" . }}-frontend
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
The current version on Roundtable is running with 5 replicas. Should this be configurable?

Reply:
I don't think so.  Based on my understanding of the app, the extra replicas will now be worker pods, and there is a single frontend pod serving images.  If I'm wrong, we'll change it then.